docs/wiki/development/building.md

```diff
@@ -281,69 +281,123 @@ vagrant up aws-amazon2015.03 --provider=aws
 vagrant ssh aws-amazon2015.03
 ```
 
-## Custom Packages
+## Building packages
 
-Package creation is facilitated by CPack. Creating a standalone custom package on any platform should be as simple as running:
+The packaging logic is now in a separate repository, [osquery-packaging](https://github.com/osquery/osquery-packaging). Packages are created in two steps:
+
+1. Create the package data from osquery/osquery
+2. Use the osquery/osquery-packaging logic to create the actual packages
+
+This approach allows maintainers to easily re-generate all the officially supported packages without going through the whole build process, which is especially useful when applying code signing from a protected machine.
+
+### Generating the package data
+
+1. Build osquery
+2. Create a destination folder and set its path in the `DESTDIR` environment variable
+3. Run the `install` target
 
 ```sh
-cmake --build . --target package
+cd build_folder
+mkdir package_data
+export DESTDIR=$(pwd)/package_data  # on Windows: `set DESTDIR=path` or `$Env:DESTDIR=path` for PowerShell
+cmake --build . --target install    # on Windows: add --config Release
 ```
+The newly created folder will include several components:
 
-Invoking the `package` target will instruct `cpack` to auto resolve all of the package
-dependencies for osquery and build a platform specific package. The package created will include several components:
-
-- The executables: `osqueryd`, `osqueryi`, and small management script `osqueryctl`
-- An osquery systemd unit on Linux (with initd script wrapper)
-- An osquery LaunchDaemon on macOS
+- The executables: `osqueryd`, `osqueryi`
 - The lenses provided by our Augeas third-party dependency
 - A default, or fall-back, OpenSSL certificate store (found within the repository)
 - The example query packs from the repository
 - Folder structures required for logging
+- Windows: small management scripts: `osqueryctl` and `manage-osqueryd.ps1`
+- Linux: systemd units, with initd script wrappers
+- macOS: a LaunchDaemon unit
+- Package control files (i.e.: deb-specific configurations, XML data required by WIX to generate MSI, etc..)
 
-What follows are instructions for directly invoking `cpack` on each platform, should
-you wish to create a more custom deployment package. For the most part this is not
-encouraged, and users should stick with leveraging the `package` target as detailed above.
+### Identifying the osquery version
 
-### On Linux
+**Linux, macOS**
 
-To create a DEB, RPM, or TGZ on Linux, CPack will attempt to auto-detect the appropriate package type.
-You may override this with the CMake `PACKAGING_SYSTEM` variable as seen in the example below.
+```sh
+cd osquery_source_folder
+git fetch --tags
+
+export OSQUERY_VERSION=$(git describe --tags --always)
+```
+
+**Windows**
+```batch
```

Comment:
Double check this file to make sure there are empty newlines between statements and code sections.